src/System.Reflection.TypeExtensions/tests/MemberBase/MethodBaseContainsGenericParameters.cs

```diff
@@ -1,227 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Reflection;
-
-using Xunit;
-
-namespace System.Reflection.Compatibility.UnitTests
-{
-    // ContainsGenericParameters
-    public class MethodBaseContainsGenericParameters
-    {
-        public class TestClass
-        {
-            public TestClass()
-            {
-            }
-
-            public TestClass(int val)
-            {
-            }
-
-            public void TestGenericMethod<T>(T p1)
-            {
-            }
-
-            public void TestMethod(int val)
-            {
-            }
-
-            public void TestMethod2(int val)
-            {
-            }
-            public void TestMethod2(int val1, float val2, string val3)
-            {
-            }
-
-            public void TestPartialGenericMethod<T>(int val, T p1)
-            {
-            }
-
-            public T TestGenericReturnTypeMethod<T>()
-            {
-                T ret = default(T);
-
-                return ret;
-            }
-        }
-
-        public class TestGenericClass<T>
-        {
-            public TestGenericClass()
-            {
-            }
-
-            public TestGenericClass(T val)
-            {
-            }
-
-            public TestGenericClass(T p, int val)
-            {
-            }
-
-            public void TestMethod(T p1)
-            {
-            }
-
-            public void TestMultipleGenericMethod<U>(U p2)
-            {
-            }
-
-            public void TestVoidMethod()
-            {
-            }
-        }
-
-        // Positive Test 1: ContainsGenericParameters should return true for open generic method
-        [Fact]
-        public void PosTest1()
-        {
-            Type type = typeof(TestClass);
-            MethodBase methodInfo = type.GetMethod("TestGenericMethod");
-            Assert.True(methodInfo.ContainsGenericParameters, "ContainsGenericParameters returns false for open generic method" + methodInfo);
```

Comment:
Oh I forgot to write this down in the PR description.

I rebased PR #11484, and git didn't delete the old member base file.

You can see the old tests here: https://github.com/dotnet/corefx/blob/master/src/System.Reflection.TypeExtensions/tests/MethodBaseTests.cs#L12